src/Kernel-Tests/TestDelayBasicSchedulerMicrosecondTicker.class.st

```diff
@@ -0,0 +1,382 @@
+"
+Please comment me using the following template inspired by Class Responsibility Collaborator (CRC) design:
```

Comment:
this comment is a "no comment" 
can you fix it before integration?

Reply:
yes, I hate that.
We'll need to review a bit Calypso UI in those kind of details.